public/pages/CreateMonitor/components/MonitorExpressions/expressions/WhereExpression.js

```diff
@@ -180,14 +180,14 @@ class WhereExpression extends Component {
   };
 
   render() {
-    const {
+    let {
```

Comment:
Would `fieldPath` or any of these parameters be changed later on?

Reply:
Good catch! This is a remnant of some experimenting I was doing. I'll refactor to `const`.